test/common/runtime/runtime_impl_test.cc

```diff
@@ -72,12 +75,20 @@ class RuntimeImplTest : public testing::Test {
     EXPECT_CALL(dispatcher, createFilesystemWatcher_())
         .WillOnce(ReturnNew<NiceMock<Filesystem::MockWatcher>>());
 
+    if (os_sys_calls_ == nullptr) {
```

Comment:
Does setup actually get called more than once in a test? Otherwise I would just ASSERT it's nullptr or just let it leak and fail the test.

Reply:
No, that was my hack so I could create this and set an EXPECT_CALL before this function creates the LoaderImpl.  A cleaner way to do it would be to have a setup() and run() function, and insert my EXPECT_CALL between the two.  Want me to do that?

Reply:
I would just remove the if check. We do this all over the place in tests. If it leaks the leak checker will find it.

Reply:
Oh, nevermind. I see. Yeah I would switch to setup() and run() if you don't mind.